docs/Easyconfigs_index.rst

```diff
@@ -0,0 +1,175 @@
+for...``"::
+
+  $ eb --search TensorFlow-2.1.0-foss-2019b
+  == found valid index for /home/example/easybuild-easyconfigs/easybuild/easyconfigs, so using it...
+ * /home/example/easybuild-easyconfigs/easybuild/easyconfigs/t/TensorFlow/TensorFlow-2.1.0-foss-2019b-Python-3.7.4.eb
+
+
+.. _easyconfigs_index_ignore:
+
+Ignoring indices (``--ignore-index``)
+-------------------------------------
+
+One potential issue with having an index in place is that it may get outdated:
+new files may have been added to the directory since the index was created or last updated.
+
+If updating the indexes is not an option (see :ref:`easyconfigs_index_update`),
+you can instruct EasyBuild to ignore any existing indices using the ``--ignore-index``
+configuration option.
+
+The only downside of this option is that searching for easyconfig files may be significantly slower.
+Any existing index files are left untouched (they will *not* be updated or removed).
+
+
+.. _easyconfigs_index_max_age:
+
+Controlling how long the index is valid (``--index-max-age``)
+-------------------------------------------------------------
+
+When creating an index file, you can specify how long the index should be considered valid.
+
+Using the ``--index-max-age`` configuration option, you can specify how long after the creation time
+the index remains valid (in seconds).
+
+By default, EasyBuild will consider index files to remain valid for 1 week (7 * 24 * 60 * 60  = 604,800 seconds).
+
+To create an index that *always* remains valid (never expires), use zero (``0``) as value for ``--index-max-age``::
+
+  $ eb --create-index --index-max-age=0 $HOME/easybuild-easyconfigs/easybuild/easyconfigs
+
+  $ head -n 2 $HOME/easybuild-easyconfigs/easybuild/easyconfigs/.eb-path-index
+  # created at: 2020-04-13 15:10:07.173191
+  # valid until: 9999-12-31 23:59:59.999999
+
+.. note:: Trust us here, December 31st 9999 is the end of times. Better get prepared.
+
+
+.. _easyconfigs_index_release:
+
+Index included with EasyBuild releases
+--------------------------------------
+
+Each EasyBuild release (since EasyBuild v4.2.0) comes with a index file for the easyconfig (and patch) files
+that are included with that release.
+
+Hence, you only need to use ``--create-index`` to create/update the index file for any additional directories
+with easyconfig files you may have on the side (and only if searching those easyconfig files is rather slow).
+
+
+.. _easyconfigs_index_should_use:
+
+Should I create an index?
+-------------------------
+
+Whether or not you should create an index file for your directories housing additional easyconfig files depends on a number of factors, including:
+
+* how often files or added and/or removed in those directories, since files listed in the index are assumed to be there and any files not included in the index will be overlooked by EasyBuild when it's searching for files;
```

Comment:
```suggestion
* how often files are added and/or removed in those directories, since files listed in the index are assumed to be there and any files not included in the index will be overlooked by EasyBuild when it's searching for files;
```